roles/openshift_service_catalog/tasks/wire_aggregator.yml

```diff
@@ -1,16 +1,82 @@
 ---
+- name: Make temp cert dir
+  command: mktemp -d /tmp/openshift-service-catalog-ansible-XXXXXX
+  register: certtemp
+  changed_when: False
+
+- name: Check for First Master Aggregator Signer cert
+  stat:
+    path: /etc/origin/master/front-proxy-ca.crt
+  register: first_proxy_ca_crt
+  changed_when: false
+  delegate_to: "{{ first_master }}"
```

Comment:
may need `run_once: true` here otherwise all three hosts will delegate the command to master in parallel and potentially stomp on each other.